dwi_ml/experiment/learning_utils.py

```diff
@@ -16,5 +21,5 @@ def compute_gradient_norm(parameters):
     for p in parameters:
         param_norm = p.grad.as_tensor.norm(2)
         total_norm += param_norm.item() ** 2
-    total_norm = total_norm ** (1. / 3)
+    total_norm = total_norm ** (1. / 2)
```

Comment:
Good catch! Luckily, this function was only used for monitoring, and did not affect any computation

Reply:
ok, good!